modules/caddypki/crypto.go

```diff
@@ -71,14 +85,14 @@ type KeyPair struct {
 }
 
 // Load loads the certificate and key.
```

Comment:
```suggestion
// Load loads the certificate chain and key.
```

I guess we doin' chains now ⛓️